docs/operation/operation.md

```diff
@@ -918,6 +918,7 @@ However, Skipper will run without this feature, if no `-passive-health-check` is
 The parameters of `-passive-health-check` option are:
 + `period=<duration>` - the duration of stats reset period
 + `min-requests=<int>` - the minimum number of requests per `period` per backend endpoint required to activate PHC for this endpoint
++ `min-drop-probabilty=<float more than/equal to 0 and less than/equal to 1>` - the minimum possible probability of unhealthy endpoint being not considered while choosing the endpoint for the given request. The same value is in fact used as minimal failed requests ratio for PHC to be enabled for this endpoint
```

Comment:
maybe simple `0.0 <= n <= 1.0` would be enough and would reduce text, same for other variables

Reply:
I mean instead of `float more than/equal to 0 and less than/equal to 1`

Reply:
`n` is assosiated with natural number, which is not the case here, I will pick up `p` instead.

Reply:
anyway I meant a variable name :)